WordPress/src/main/java/org/wordpress/android/localcontentmigration/UserFlagsProviderHelper.kt

```diff
@@ -66,7 +66,6 @@ class UserFlagsProviderHelper @Inject constructor(
         DeletablePrefKey.RECENTLY_PICKED_SITE_IDS.name,
         UndeletablePrefKey.THEME_IMAGE_SIZE_WIDTH.name,
         UndeletablePrefKey.BOOKMARKS_SAVED_LOCALLY_DIALOG_SHOWN.name,
-        UndeletablePrefKey.IMAGE_OPTIMIZE_PROMO_REQUIRED.name,
```

Comment:
@SiobhyB If I'm not wrong, this is used to determine which preferences should be migrated from the WordPress app. I presume that removing it has no effect, i.e. the flag won't be migrated, but I propose adding this case to the testing instructions.

Reply:
I couldn't manage to test the migration with pre-alpha builds. Based on the code, this change seems safe but we could double-check with the next beta version.

Reply:
Thanks for the note, Carlos! I'll make sure to double-check this with the next beta version.